source/expressions.tex

```diff
@@ -319,7 +319,7 @@
 that does not denote an object of type \cv{}~\keyword{U} within its lifetime,
 the behavior is undefined.
 \begin{note}
-Unlike in C, \Cpp{} has no accesses of class type.
+Unlike in C, \Cpp{} has no accesses through a glvalue of class type.
```

Comment:
Hm, but C also has no notion of "glvalue", so even though this change has made the language more precise on one way, it's also intruduced a kind of non-sequitur comparison. Maybe we need to say this a bit more verbosely?

Reply:
Neither C has notion of class type. How verbose you want this to become?

Reply:
Well, if we're already interested in clarifying this, we should try to do the best possible job. Can you think of anything that would help?

Reply:
Maybe: "In C, entire structs can be accessed, e.g. using assignment. In contrast, \Cpp{} has no notion of accessing an entire struct through an lvalue of class type."

Reply:
That's the spirit. Thanks, Jens!